platform/security/platform-security-core-api/src/main/java/ddf/security/permission/Permissions.java

```diff
@@ -47,10 +47,14 @@ private Permissions() {
                     String[] parts = perm.split("=");
                     if (parts.length == 2) {
                         String attributeName = parts[0];
-                        String attributeValue = parts[1];
-                        String[] attributeValues = attributeValue.split(",");
-                        permissions.put(attributeName,
-                                new HashSet<>(Arrays.asList(attributeValues)));
+                        Scanner s = new Scanner(parts[1]).useDelimiter(",");
```

Comment:
❗️  This works but it's less efficient than using a compiled `Pattern`.

I just ran a comparison of the `Scanner` code vs. the following code over 10K runs and the `Scanner` takes about three times longer. It's even more pronounced - about four-to-five times slower - when I make the `Pattern` a static final.
```java
        Pattern pattern = Pattern.compile(",");

        return pattern.splitAsStream(input)
                .map(String::trim)
                .collect(Collectors.toSet());
```